### PR TITLE
[webpack.config.js] update dependencies for copy-webpack-plugin

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Updates package.json for latest version of copy-webpack-plugin.

#### Testing instructions (if applicable)

1. make clean
2. make dev
3. navigate around Loris

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -125,11 +132,12 @@ function lorisModule(mname, entries, override=false) {
...
       'react-dom': 'ReactDOM',
     },
-    node: {
-      fs: 'empty',
-    },
     devtool: 'source-map',
-    plugins: [],
+    plugins: [
+      new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('development'),
```
Comment (maltheism) [inline webpack.config.js:138]: Solves the following error in the browser:
`Webpack: Bundle.js - Uncaught ReferenceError: process is not defined`
See https://webpack.js.org/migrate/5/ and https://stackoverflow.com/questions/41359504/webpack-bundle-js-uncaught-referenceerror-process-is-not-defined

File: webpack.config.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
   minimizer: [
-    new TerserPlugin({
-      cache: true,
-      parallel: true,
-      terserOptions: {
-        compress: false,
-        ecma: 6,
-        mangle: false,
-      },
-      sourceMap: true,
```
Comment (maltheism) [inline webpack.config.js:17]: I believe that `devtool: 'source-map'` handles the `sourceMap: true` from looking at the latest terser-webpack-plugin documentation.
```
[webpack-cli] Invalid options object. Terser Plugin has been initialized using an options object that does not match the API schema.
54
 - options has an unknown property 'sourceMap'. These properties are valid:
55
   object { test?, include?, exclude?, terserOptions?, extractComments?, parallel?, minify? }
```

File: webpack.config.js [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
 const TerserPlugin = require('terser-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
+const webpack = require('webpack');
 const path = require('path');
 const fs = require('fs');
 
 const optimization = {
   minimizer: [
-    new TerserPlugin({
-      cache: true,
```
Comment (maltheism) [inline webpack.config.js:10]:
```
5.0.0 (2020-10-14) ⚠ BREAKING CHANGES
* webpack@4 no longer supported
* removed the cache option (respect the cache option by default from webpack)
* removed the cacheKeys option (respect the cache option by default from webpack)
* removed the sourceMap option (respect the devtool option by default from webpack)
Bug Fixes
* reduce deps
* performance improvement
```

**Comment by maltheism** [comment]
I rebased. Hi @driusan this PR is ready again.

**Review by driusan — APPROVED** [review]

**Pull request merged**